README.md

```diff
@@ -122,27 +121,54 @@ app_1        | * Environment: development
 app_1        | * Listening on tcp://0.0.0.0:3000
 app_1        | Use Ctrl-C to stop
 ```
-Once the app is running, you can connect to it by navigating to `localhost`. Please note that trying to connect to the app at `localhost` before it is ready will result in `502` Bad Gateway error.
+Once the app is running, you can connect to it by navigating to `localhost`. Please note that trying to connect to the app at `localhost` before it is ready will result in `502` Bad Gateway error, so be sure to check the logs first.
 
 **Testing**
 
 If you would like to run commands against your app service, you can do that with the following command (using rubocop as an example): 
 
 ```
-docker-compose exec app bundle exec rubocop app config db lib spec --safe-auto-correct
+./script/test-command.sh bundle exec rubocop app config db lib spec --safe-auto-correct
 ```
 
 Or to run a particular spec:
 
 ```
-docker-compose exec app bundle exec rspec <your-spec-file>
+./script/test-command.sh bundle exec rspec <your-spec-file>
 ```
+**Running migrations**
+
+In cases where you want to create a migration in the context of your current development, you can use the following command:
+
+```
+docker-compose exec app rails g migration <your migration>
+```
+To run the migration, type:
+
+```
+docker-compose exec app bundle exec rake db:migrate
```

Comment:
If there is a pending migration, and the app is currently stopped, will this work?

iirc (haven't tested), the app won't start with a pending migration, so you can't exec things (such as running the migration)?

Reply:
I think in that case I would use the `restart-app` script. I've added some documentation to make that clear!